ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/Widgets/HorizontalBarChart.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { Chart, ChartAxis, ChartBar, ChartGroup, ChartLabelProps } from '@patternfly/react-charts';
+
+import { LinkableChartLabel } from 'Components/PatternFly/Charts/LinkableChartLabel';
+import useResizeObserver from 'hooks/useResizeObserver';
+import {
+    defaultChartBarWidth,
+    defaultChartHeight,
+    patternflySeverityTheme,
+} from 'utils/chartUtils';
+
+import { getBarColor } from './ColorsForCompliance';
+import { PassingRateData } from '../types';
+
+const labelLinkCallback = ({ datum }: ChartLabelProps, data: PassingRateData) => {
+    return typeof datum === 'number' ? data[datum - 1]?.link ?? '' : '';
+};
+
+type HorizontalBarChartProps = {
+    passingRateData: PassingRateData;
+};
+
+function HorizontalBarChart({ passingRateData }: HorizontalBarChartProps) {
+    const [widgetContainer, setWidgetContainer] = useState<HTMLDivElement | null>(null);
+    const widgetContainerResizeEntry = useResizeObserver(widgetContainer);
+
+    return (
+        <div ref={setWidgetContainer}>
+            <Chart
+                animate={{ duration: 300 }}
+                domainPadding={{ x: [20, 20] }}
+                height={defaultChartHeight}
+                width={widgetContainerResizeEntry?.contentRect.width}
+                padding={{
+                    top: 0,
+                    left: 150,
+                    right: 50,
+                    bottom: 30,
+                }}
+                theme={patternflySeverityTheme}
+            >
+                <ChartAxis
+                    tickLabelComponent={
+                        <LinkableChartLabel
+                            linkWith={(props) => labelLinkCallback(props, passingRateData)}
+                        />
+                    }
+                />
+                <ChartAxis
+                    tickValues={[0, 50, 100]}
+                    tickFormat={['0', '50%', '100%']}
+                    padding={{ bottom: 10 }}
+                    dependentAxis
+                />
+                <ChartGroup horizontal>
+                    {passingRateData.map(({ name, passing }) => (
+                        <ChartBar
+                            key={name}
+                            barWidth={defaultChartBarWidth}
+                            data={[{ x: name, y: passing }]}
+                            labels={({ datum }) => `${Math.round(parseInt(datum.y, 10))}%`}
```

Comment:
Has `parseInt` already truncated the value of `y` before `Math.round` gets its turn?

Reply:
You're right. Depending on if the if `y` is a decimal I should either use parseFloat or just strip the Math.round portion. 

`Math.round(parseInt(datum.y, 10.6)) // 10`
`Math.round(parseFloat(datum.y, 10.6)) // 11`

but that brings up the question, how would we decide what to display for 99.9 or 0.1? Probably not super important to determine here but an interesting question nonetheless